api/cms.go

```diff
@@ -0,0 +1,72 @@
+package api
+
+import (
+	"errors"
+	"github.com/spf13/viper"
+	"gopkg.in/gin-gonic/gin.v1"
+	"os"
+)
+
+func CMSPerson(c *gin.Context) {
+	id := c.Query("id")
+	fbLang := c.DefaultQuery("fbLang", "en")
+	lang := c.DefaultQuery("lang", fbLang)
+
+	assets := viper.GetString("cms.assets")
```

Comment:
move to common.go as some sort of global

Reply:
done